opentelemetry-stackdriver/src/lib.rs

```diff
@@ -657,3 +652,223 @@ pub enum MonitoredResource {
+        );
+
+        // 	userAgentAttribute  = "http.user_agent"
+        attributes
+            .insert(semcov::trace::HTTP_USER_AGENT.string("CERN-LineMode/2.15 libwww/2.17b3"));
+
+        // 	statusCodeAttribute = "http.status_code"
+        attributes.insert(semcov::trace::HTTP_STATUS_CODE.i64(200));
+
+        // 	statusCodeAttribute = "http.route"
+        attributes.insert(semcov::trace::HTTP_ROUTE.string("/webshop/articles/:article_id"));
+
+        // 	serviceAttribute    = "service.name"
+        attributes.insert(semcov::resource::SERVICE_NAME.string("Test Service Name"));
+
+        let actual: Attributes = attributes.into();
+
+        assert_eq!(actual.attribute_map.len(), 8);
+        assert_eq!(actual.dropped_attributes_count, 0);
+        assert_attribute_map_contains(
+            &actual,
+            "/http/host",
+            AttributeValue::from(Value::String("example.com:8080".into())),
+        );
+        assert_attribute_map_contains(
+            &actual,
+            "/http/method",
+            AttributeValue::from(Value::String("POST".into())),
+        );
+        assert_attribute_map_contains(
+            &actual,
+            "/http/path",
+            AttributeValue::from(Value::String("/path/12314/?q=ddds#123".into())),
+        );
+        assert_attribute_map_contains(
+            &actual,
+            "/http/route",
+            AttributeValue::from(Value::String("/webshop/articles/:article_id".into())),
+        );
+        assert_attribute_map_contains(
+            &actual,
+            "/http/url",
+            AttributeValue::from(Value::String(
+                "https://example.com:8080/webshop/articles/4?s=1".into(),
+            )),
+        );
+        assert_attribute_map_contains(
+            &actual,
+            "/http/user_agent",
+            AttributeValue::from(Value::String("CERN-LineMode/2.15 libwww/2.17b3".into())),
+        );
+        assert_attribute_map_contains(
+            &actual,
+            "/http/status_code",
+            AttributeValue::from(Value::I64(200)),
+        );
+        assert_attribute_map_contains(
+            &actual,
+            "g.co/gae/app/module",
+            AttributeValue::from(Value::String("Test Service Name".into())),
+        );
+    }
+
+    #[test]
+    fn test_attributes_mapping_http_target() {
+        let capacity = 10;
+        let mut attributes = trace::EvictedHashMap::new(capacity, 0);
+
+        //	hostAttribute       = "http.target"
+        attributes.insert(semcov::trace::HTTP_TARGET.string("/path/12314/?q=ddds#123"));
+
+        let actual: Attributes = attributes.into();
+
+        assert_eq!(actual.attribute_map.len(), 1);
+        assert_eq!(actual.dropped_attributes_count, 0);
+        assert_attribute_map_contains(
+            &actual,
+            "/http/path",
+            AttributeValue::from(Value::String("/path/12314/?q=ddds#123".into())),
+        );
+    }
+
+    fn assert_attribute_map_contains(
```

Comment:
I'd prefer to just replace this with the inner `assert_eq!()` call. It doesn't seem useful to separately check whether the value is in there before asserting that the `get()` is `Some()`?